src/addresses/Address-details/AddressDetails.component.tsx

```diff
@@ -2,16 +2,16 @@ import { FC, useContext, useState } from 'react';
 import { Thead, Tr, Th, Tbody } from '@patternfly/react-table';
 import { Table } from '@patternfly/react-table/deprecated';
 import {
-  Spinner,
-  Bullseye,
   Pagination,
   PaginationVariant,
   SearchInput,
-  Alert,
   Title,
   ToolbarContent,
   Toolbar,
   ToolbarItem,
+  Alert,
+  Bullseye,
+  Spinner,
```

Comment:
I would prefer if we wouldn't introduce any changes in the file, as this is unrelated to the current PR.